### PR TITLE
fmod.md for PHP

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

This is my attempt to create a reference for the fmod() function in PHP. Motivation stems from the sustained momentum of contributing to docs and sticking with the primary goal of helping others.

### Type of Change

- Adding a new entry

### Checklist

- [ /] All writings are my own.
- [ /] My entry follows the Codecademy Docs style guide.
- [ /] My changes generate no new warnings.
- [ /] I have performed a self-review of my own writing and code.
- [ /] I have checked my entry and corrected any misspellings.
- [ /] I have made corresponding changes to the documentation if needed.
- [ /] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ /] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ /] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/fmod/fmod.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - 'Arithmetic'
+  - 'Encoding'
+  - 'Functions'
+  - 'Math'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`fmod()`** function returns the remainder (Modulo) of x/y. It stands for Floating Modulo and calculates the Modulo of any division which may contain a `float` as both dividend and divisor.
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/fmod/fmod.md:17]:
```suggestion
The **`fmod()`** function returns the remainder (modulo) of *x/y*. The name stands for "floating modulo" and the function calculates the modulo of any division which may contain a `float` as both dividend and divisor.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@arisdelacruz Just a few minor edits. Please update, and tag me when complete.

**Comment by arisdelacruz** [comment]
@caupolicandiaz 

just completed

thanks, buddy

**Review by caupolicandiaz — APPROVED** [review]
@arisdelacruz Looks good. I'm passing this on to a second review.

**Review by SSwiniarski — APPROVED** [review]
@arisdelacruz Looks good.

**Comment by SSwiniarski** [comment]
@arisdelacruz Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/php/math-functions/fmod

image

**Pull request merged**